Sources/Core/JavaIR.swift

```diff
@@ -133,24 +171,34 @@ public struct JavaIR {
 
         func render() -> [String] {
             let implementsList = implements?.joined(separator: ", ") ?? ""
-            let implementsStmt = implementsList == "" ? "" : "implements \(implementsList)"
-
-            var propertiesStrings = [String]()
-            for propertyBatch in properties {
-                for property in propertyBatch {
-                    propertiesStrings.append(property.render())
-                }
-                propertiesStrings.append("") // Add an empty line between each batch
-            }
+            let implementsStmt = implementsList == "" ? "" : " implements \(implementsList)"
```

Comment:
maybe change `implementsList == ""` to `implementsList.isEmpty`?

Reply:
Done.